src/components/Home.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
Creating a Home component, this way we have a component to represent the home page, which contains the search.